MovieTime/MovieTime/Service/MTParser.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  MTParser.swift
+//  MovieTime
+//
+//  Created by Enrique Melgarejo on 09/01/18.
+//  Copyright © 2018 Choynowski. All rights reserved.
+//
+
+import Foundation
+
+final class MTParser<T> where T: Decodable {
```

Comment:
There are many reasons why I do not use third-party libraries in this context:
- Sometimes we just needs a part of a library and it may be more cost-effective, and more maintainable with time, to develop your own.
- Sometimes the library becomes an obsolete project.

There are many libraries that deal with data parsing and network handling and in all cases where I decided to use a framework I have had problems with maintenance, growth, version update and lack of documentation.

Reply:
there are several great, stable and community-maintained frameworks out there, you just need to pick your choices wisely :) if you see recent commits and at least a few thousands of stars, you should be good to go. Given that the time we allow for this code challenge is very small, some frameworks come in very handy and picking good ones show us you're up-to-date with what the community has been doing and that you know where to better dedicate your time. But of course, we were able to evaluate your low-level knowledge, which is great for other reasons :)